.gitignore

```diff
@@ -2,6 +2,10 @@
 dist
 storybook-static
 
+# Generated files
+/src/assets/**/*.twig
```

Comment:
I take it these are our SVG to Twig generated files? Might be nice to make a note so it's clear what exactly we're generating and ignoring there (since generally I'd assume twig files should not be ignored)